loop-solutions.js

```diff
@@ -0,0 +1,79 @@
+// 1)
+
+let num = 100;
+
+while (num >= 1) {
+  console.log(num);
+  num--;
+};
+
+// 1.1)
+
+for (let num = 100; num >= 1; num--) {
+  console.log(num);
+};
+
+// 2)
+
+let num = 0;
+
+while (num <= 100) {
+  if (num % 2 === 0) {
+    console.log(num + ' is even');
+  } else {
+    console.log(num + ' is odd');
+  };
+  num++;
+};
+
+// 2.1)
+
+for (let num = 0; num <= 100; num ++) {
+  if (num % 2 === 0) {
+    console.log(num + ' is even');
+  } else {
+    console.log(num + ' is odd');
+  }
+}
+
+// 3)
+
```

Comment:
Was hoping the log would be that little string sentence from the exercise, but it's okay. It's clear you understand how to write loops well.